### DISCUSSION TITLE
npm run dev没问题，npm run build报错了

### DISCUSSION THREAD

**Discussion by FEliuze** [vbenjs/vue-vben-admin#1292] [Q&A]
node版本： v12.16.0
终端： mac
错误 ：
error during build:
Error: [vite]: Rollup failed to resolve import "/Users/liuze/code/xdf/sas-front/node_modules/_/ant-design-vue/es/config-provider/style/index" from "src/App.vue".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`

**Comment by myltx** [comment]
你好，你的这个问题解决了吗？

**Comment by qiekenol** [comment]
node升级一下
